app/models/organization.rb

```diff
@@ -21,6 +21,8 @@ class Organization < ApplicationRecord
 
   delegate :email, to: :owner
 
+  scope :discoverable, -> { where(privacy_setting: :public).or(where(privacy_setting: :restricted)) }
```

Comment:
again, with `enum` you can just do `Organization.not_privacy_setting_private` instead of a custom scopes